presto-hive/src/main/java/io/prestosql/plugin/hive/util/InternalHiveSplitFactory.java

```diff
@@ -91,17 +92,31 @@ public String getPartitionName()
         return partitionName;
     }
 
-    public Optional<InternalHiveSplit> createInternalHiveSplit(LocatedFileStatus status, boolean splittable)
+    public Optional<InternalHiveSplit> createInternalHiveSplit(LocatedFileStatus status, boolean splittable, Optional<DeleteDeltaLocations> deleteDeltaLocations)
     {
-        return createInternalHiveSplit(status, OptionalInt.empty(), splittable);
+        return createInternalHiveSplit(status, OptionalInt.empty(), splittable, deleteDeltaLocations);
     }
 
     public Optional<InternalHiveSplit> createInternalHiveSplit(LocatedFileStatus status, int bucketNumber)
     {
-        return createInternalHiveSplit(status, OptionalInt.of(bucketNumber), false);
+        return createInternalHiveSplit(status, OptionalInt.of(bucketNumber), false, Optional.empty());
     }
 
-    private Optional<InternalHiveSplit> createInternalHiveSplit(LocatedFileStatus status, OptionalInt bucketNumber, boolean splittable)
+    public Optional<InternalHiveSplit> createInternalHiveSplit(FileStatus status, BlockLocation[] blockLocations)
```

Comment:
I guess this doesn't belong to `Add DELETE_DELTA locations in HiveSplit` commit.

Reply:
Yes, this came in along with the original files change.
Removing.